package.json

```diff
@@ -37,6 +37,7 @@
     }
   },
   "devDependencies": {
+    "@babel/plugin-transform-runtime": "^7.9.0",
```

Comment:
is this new plugin necessary? I think that `babel-env` should provide same features.